### PR TITLE
Resource grid components

### PR DESCRIPTION
**Pull request by woosie10**
This PR moves the resource grid components from MyCloud and improves them. It also adds a resource grid story to Storybook, as well as tests.

### CODE REVIEW

File: src/stylesheets/_resource_empty_tile.scss [hunk 1]
```diff
@@ -0,0 +1,15 @@
+.resource-empty-state {
+    text-align: center;
+
+    .resource-empty-state__text {
```
Comment (ImreBognarUltimaker) [inline src/stylesheets/_resource_empty_tile.scss]: Must these be nested?

File: src/stylesheets/_resource_empty_tile.scss [hunk 2]
```diff
@@ -0,0 +1,15 @@
+.resource-empty-state {
```
Comment (ImreBognarUltimaker) [inline src/stylesheets/_resource_empty_tile.scss]: Shouldn't this file be named _resource_empty_state.scss ?

File: src/stylesheets/_sub_resource_list.scss [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    }
+
+    .pill {
+        height: 3rem;
+        line-height: 3rem;
+    }
+
+    .more-pill .pill {
+        .more-pill__text {
+            display: inline-block;
```
Comment (ImreBognarUltimaker) [inline src/stylesheets/_sub_resource_list.scss]: I think using only .more-pill__text here should be enough.

File: src/stylesheets/_resource_empty_tile.scss (hunk 1 above)
Comment (woosie10) [reply src/stylesheets/_resource_empty_tile.scss]: updated

File: src/stylesheets/_resource_empty_tile.scss (hunk 2 above)
Comment (woosie10) [reply src/stylesheets/_resource_empty_tile.scss]: updated

File: src/stylesheets/_sub_resource_list.scss (hunk 3 above)
Comment (woosie10) [reply src/stylesheets/_sub_resource_list.scss]: updated

**Pull request merged**